sdk/resourceconnector/Azure.ResourceManager.ResourceConnector/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0 (2023-08-25)
```

Comment:
This is new package, so we need follow this [policy](https://azure.github.io/azure-sdk/policies_releases.html#beta-releases-and-stable-graduation) to release a beta first then stable after one month.

Reply:
Updated this accordingly